proxy/connector.go

```diff
@@ -0,0 +1,79 @@
+package proxy
+
+import (
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/providers"
+)
+
+// Connector is a Hubspot connector.
+type Connector struct {
+	Client *common.JSONHTTPClient
+
+	provider      providers.Provider
+	providerInfo  *providers.ProviderInfo
+	substitutions *map[string]string
```

Comment:
This is only needed at construction time, right? Seems wasteful to keep it alive in memory if it'll only ever get used once.